src/index.html

```diff
@@ -17,6 +30,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
```

Comment:
The `href` attribute is set to `#`. Please replace it with an actual URL to ensure proper navigation functionality.